Tutorials/LinearSolvers/MultiComponent/MCNodalLinOp.cpp

```diff
@@ -62,6 +66,10 @@ void MCNodalLinOp::Diag (int amrlev, int mglev, MultiFab& a_diag)
 	domain.grow(-1); // Shrink domain so we don't operate on any boundaries
 	const Real* DX = m_geom[amrlev][mglev].CellSize();
 
+        amrex::AsyncArray<Real> aa(coeff.data(), coeff.size());
+        amrex::Real const* dcoeff = aa.data();
+        const int N = getNComp();
+
```

Comment:
Alignment looks a bit off:
```suggestion
      amrex::AsyncArray<Real> aa(coeff.data(), coeff.size());
      amrex::Real const* dcoeff = aa.data();
      const int N = getNComp();

```